operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -175,6 +185,28 @@ The Technical Product Writer/s form part of the PR review/approval process. When
 
 The [Documentation Style guide](https://docs.mattermost.com/guides/core.html#documentation-style-guide) and [UI text guidelines](https://docs.mattermost.com/process/documentation-UItext-guidelines.html).
 
+## Customer Education Responsibilities
+
+### Identifying and ranking gaps and pain points among customers
+
+When it comes to identifying gaps and pain points, ranking them is important in order to correctly prioritize.
+
+#### Identification
+
+- Monitor support tickets and get regular feedback from Support Engineers through the support channel
+- Ensure information flow with Solutions Architects and Customer Success Managers e.g. by receiving a regular report
+- Use already existing structures between Customer Support and Customer Success to avoid overhead; Customer Support is constantly sharing information with Customer Success Managers
+
+#### Ranking
+
+- Use existing metrics from the ticket system like priority and customer revenue to rank identified issues
+- Compare Customer Support time put into issues to generated revenue for high impact
+
+#### What is missing
+
+- Direct information flow with CSM and SA
+- Statistics about money spent to money made on solved issues
```

Comment:
Would this be public knowledge or internal? Might be something to consider if it's stated publicly but not available publicly.

Reply:
great question, how would I find out @justinegeffen

Reply:
I'd definitely confirm with @jasonblais.

Reply:
This section was actually removed, resolving this conversation